tuiter/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
```

Comment:
good job on put all those in gitignore

Reply:
should we share the build so when we download there are no version conflict?

Reply:
The main branch is public so anyone can download it. So it is shared. To create branches from the main you need to become a collaborator. So once they become collaborator they can get main branch and make their own branch to change and send a pull request to have their change push to main.

Reply:
you should pull from main branch so you have the latest version of the main branch.

Reply:
oh wait. the build file isn't needed since you can run build anytime you want without the build file.